dashboard/app/assets/stylesheets/scaffolds.css.scss

```diff
@@ -1,6 +1,8 @@
+@import "color";
+
 body {
-  background-color: #fff;
-  color: #333;
+  background-color: $white;
+  color: $dark_charcoal;
```

Comment:
I think `$dark_charcoal` might be too light, and sacrifice readability on bad monitors:

Before:

![screen shot 2016-08-09 at 3 56 21 pm](https://cloud.githubusercontent.com/assets/413693/17536611/0dfb4aa2-5e4b-11e6-9171-634d6f027f7d.png)

After:

![screen shot 2016-08-09 at 3 56 28 pm](https://cloud.githubusercontent.com/assets/413693/17536619/13017ecc-5e4b-11e6-90e1-2badf04eaded.png)